src/WeatherData.js

```diff
@@ -0,0 +1,39 @@
+import { TableCell, Table, TableRow, Button } from "@mui/material";
+import React from "react";
+
+export default class WeatherData extends React.Component {
+  render() {
+    const { weatherData, handleShowForecast } = this.props;
+    const buttonState = [];
```

Comment:
this array seems odd to me. Since this is an array of booleans, shouldn't there only be a boolean instead an array of it? Shouldn't we just keep track of clicked or not clicked for example? Definitely a bit confusing :)